panels/models.py

```diff
@@ -99,7 +99,10 @@ class AssignedPanelist(MagModel):
     event_id    = Column(UUID, ForeignKey('event.id', ondelete='cascade'))
 
     def __repr__(self):
-        return '<{self.attendee.full_name} panelisting {self.event.name}>'.format(self=self)
+        if self.attendee:
+            return '<{self.attendee.full_name} panelisting {self.event.name}>'.format(self=self)
+        else:
+            return super(AssignedPanelist, self).__repr__()
```

Comment:
I had to do this to prevent `__repr_`_ errors when creating a new instance of this model, since the relationship doesn't have a chance to exist right when we create a new AssignedPanelist object.